src/main/scala/uk/gov/nationalarchives/notifications/messages/EventMessages.scala

```diff
@@ -88,4 +89,16 @@ object EventMessages {
       }
     }
   }
+
+  implicit val exportStatusEventMessages: Messages[ExportStatusEvent] = new Messages[ExportStatusEvent] {
+    override def email(incomingEvent: ExportStatusEvent): Option[Email] = {
+      println(s"Skipping email for export complete message for consignment ${incomingEvent.consignmentId}")
```

Comment:
Can this be a proper log message, so we can apply log levels to it?

Reply:
I've added in a logger and changed this to `log.info`